src/main/resources/docs/pebble-templates.md

```diff
@@ -207,14 +207,14 @@ call.render("index", mapOf("name" to "Jane", "address" to address))
 <a name="sharing-view-data-on-httpcall-object"></a>
 #### [Sharing view data on HttpCall object](#sharing-view-data-on-httpcall-object)
 
-A handy method, aptly named `shareWithView(key: String, value: Any?)`, can be used to share some data
+A handy method, aptly named `share(pair: Pair<String, Any?>, vararg pairs: Pair<String, Any>)`, can be used to share some data
 with the templates from anywhere an `HttpCall` is accessible — from a helper function, a private
 function within a controller, or from within the [ValidationGuard](/docs/validation).
 
 ```kotlin
 
 // anywhere from where a call is accessible
-call.shareWithView("valuation", 25000000)
+call.share(Pair("valuation", 25000000))
```

Comment:
A more idiomatic way is `call.share("valuation" to 2500000)`

Reply:
@ashokgelal - I tried that initially - but it wouldn't let me. Had to include 'Pair'...

Reply:
That's weird. It works for me: https://dl.dropbox.com/s/tofp8wf3rzv3xwv/Screenshot%202020-03-29%2021.51.28.png?dl=0

Reply:
@ashokgelal Very weird - I get this error when I use sans 'Pair' (Alpas v 0.16.1.) - Type mismatch: inferred type is String but Pair<String, Any?> was expected.